docs/table.clone-locked-metatables.md

```diff
@@ -0,0 +1,73 @@
+# Feature name
+
+## Summary
+
+Allow `table.clone` to copy tables with locked metatables.
+
+## Motivation
+
+As proposed by the [`table.clone` RFC](function-table-clone.md), the function `table.clone` cannot create shallow copies of a table if it has a locked metatable.
+Due to this ugly limitation, it is extremely un-ergonomic to create shallow copies of arbitrary tables.
```

Comment:
Please refrain from using such language in the text of the RFC.

Reply:
Done